electron/src/electron-main.js

```diff
@@ -0,0 +1,292 @@
+'use strict'
+
+const { app, Menu, BrowserWindow, shell, session, dialog } = require('electron');
+const childProcess = require('child_process');
+const path = require('path')
+const url = require('url');
+
+// This adds refresh and devtools console keybindings
+// Page can refresh with cmd+r, ctrl+r, F5
+// Devtools can be toggled with cmd+alt+i, ctrl+shift+i, F12
+require('electron-debug')({enabled: true, showDevTools: false});
+require('electron-context-menu')({});
+
+global.eval = function() { throw new Error('bad!!'); }
+
+app.commandLine.appendSwitch('ssl-version-fallback-min', 'tls1.2');
+app.commandLine.appendSwitch('--no-proxy-server');
+app.setAsDefaultProtocolClient('skycoin');
+
+// Keep a global reference of the window object, if you don't, the window will
+// be closed automatically when the JavaScript object is garbage collected.
+let win;
+
+var server = null;
+const serverPort= 8412;
+
+// It is only possible to make connections to hosts that are in this list.
+var allowedHosts = new Map();
+allowedHosts.set('127.0.0.1:' + serverPort, true);
+allowedHosts.set('api.coinmarketcap.com', true);
+allowedHosts.set('api.github.com', true);
+
+// Indicates if the URLs of the nodes were already added to allowedHosts.
+var nodesAllowed = false;
```

Comment:
this var is never `true`

Reply:
The variable was deleted because now the procedure is different